### PR TITLE
Feat/init confs

### PR DESCRIPTION
**Pull request by WilliGautier**
Update initial conf by:

- Removing link to demo
- Fixing local arlas server address
- Removing link to eo templates

### CODE REVIEW

File: arlas/cli/cli.py [hunk 1]
```diff
@@ -42,19 +42,16 @@ def init(
...
-                "demo": ARLAS(server=Resource(location="https://demo.cloud.arlas.io/arlas/server", headers={"Content-Type": "application/json"})),
                 "local": ARLAS(
-                    server=Resource(location="http://localhost/server", headers={"Content-Type": "application/json"}),
+                    server=Resource(location="http://localhost/arlas", headers={"Content-Type": "application/json"}),
                     persistence=Resource(location="http://localhost/persist", headers={"Content-Type": "application/json"}),
                     elastic=Resource(location="http://localhost:9200", headers={"Content-Type": "application/json"}),
                     allow_delete=True
                 )
             },
             mappings={
```
Comment (WilliGautier) [inline arlas/cli/cli.py:52]: Should I remove totally the `mappings` and `models` sections or leave it empty to know where to modify it?

File: arlas/cli/cli.py [hunk 2]
```diff
@@ -42,19 +42,16 @@ def init(
...
-                "arlas_eo": Resource(location="https://raw.githubusercontent.com/gisaia/ARLAS-EO/master/mapping.json")
             },
             models={
-                "arlas_eo": Resource(location="https://raw.githubusercontent.com/gisaia/ARLAS-EO/master/collection.json")
             }
         )
         Configuration.save(variables["configuration_file"])
```
Comment (WilliGautier) [inline arlas/cli/cli.py:57]: The line under:

> Warning : no configuration file found, we created an empty one for you 

is not totally true as the created configuration contains a local conf. Do we want to change it ?

File: arlas/cli/cli.py (hunk 1 above)
Comment (QuCMGisaia) [reply arlas/cli/cli.py:52]: I think it can be interesting to leave it, to tell the user that this option exists

File: arlas/cli/cli.py (hunk 2 above)
Comment (QuCMGisaia) [reply arlas/cli/cli.py:57]: Yeah, saying "We created a default one that is compatible with the simple exploration stack" is more accurate

**Review by QuCMGisaia — APPROVED** [review]

**Review by sylvaingaudan — APPROVED** [review]

**Pull request merged**